stl/inc/__msvc_int128.hpp

```diff
@@ -29,64 +29,86 @@ _STL_DISABLE_CLANG_WARNINGS
 
 _STD_BEGIN
 
+#if defined(_M_X64) && !defined(_M_ARM64EC)
+#define _STL_128_INTRINSICS 1
+#ifdef __clang__ // clang doesn't have _udiv128 / _div128
+#define _STL_128_DIV_INTRINSICS 0
+#else // ^^^ Clang / other vvv
+#define _STL_128_DIV_INTRINSICS 1
+#endif
+#else // ^^^ X64 / other vvv

```

Comment:
Nitpicks: The `#endif` should be commented as this is densely packed preprocessor logic, and we conventionally say `x64` in lowercase.